app/uk/gov/hmrc/eusubsidycompliancefrontend/views/models/FinancialDashboardSummary.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eusubsidycompliancefrontend.views.models
+
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.types.Sector.Sector
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.types.{IndustrySectorLimit, Sector, SubsidyAmount}
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.{Undertaking, UndertakingSubsidies}
+import uk.gov.hmrc.eusubsidycompliancefrontend.util.TaxYearSyntax._
+
+import java.time.LocalDate
+
+case class FinancialDashboardSummary(
+  overall: OverallSummary,
+  taxYears: Seq[TaxYearSummary] = Seq.empty
+)
+
+case class OverallSummary(
+  startYear: Int,
+  endYear: Int,
+  hmrcSubsidyTotal: SubsidyAmount,
+  nonHmrcSubsidyTotal: SubsidyAmount,
+  sector: Sector,
+  sectorCap: IndustrySectorLimit,
+) {
+  def total: SubsidyAmount = SubsidyAmount(hmrcSubsidyTotal + nonHmrcSubsidyTotal)
+  def allowanceRemaining: SubsidyAmount = SubsidyAmount(sectorCap - total)
+}
+
+case class TaxYearSummary(
+  startYear: Int,
+  hmrcSubsidyTotal: SubsidyAmount,
+  nonHmrcSubsidyTotal: SubsidyAmount,
+) {
+  def total: SubsidyAmount = SubsidyAmount(hmrcSubsidyTotal + nonHmrcSubsidyTotal)
+  def endYear: Int = startYear + 1
+}
+
+object FinancialDashboardSummary {
+
+  // Fallback values should no sector be defined on the undertaking.
+  private val DefaultSectorLimits = Map(
+    Sector.agriculture -> IndustrySectorLimit(30000.00),
+    Sector.aquaculture -> IndustrySectorLimit(20000.00),
+    Sector.other -> IndustrySectorLimit(200000.00),
+    Sector.transport -> IndustrySectorLimit(100000.00),
+  )
+
+  // Generates summarised data to populate the financial dashboard page.
+  // All currency amounts in EUR.
```

Comment:
I think it's helpful and want then to stay

Reply:
I think this comment is needed since the types don't tell you what the currency is. There's definitely some value in these situations and perhaps it's a clue that we need to refactor this and introduce a currency type.

Reply:
👌